app/models/question/currency.rb

```diff
@@ -5,6 +5,8 @@ class Question::Currency < Question
 
   attribute :settings, :json, default: -> { assign_default_values('settings') }
 
+  validates :accepted_answers, absence: true
```

Comment:
Can we remove this line and define this in the main `Question` model with a condition?